internal/access/group.go

```diff
@@ -84,6 +88,36 @@ func DeleteGroup(c *gin.Context, id uid.ID) error {
 	return data.DeleteGroups(db, selectors...)
 }
 
+func checkIdentitiesInList(db *gorm.DB, ids []uid.ID) ([]uid.ID, error) {
+	contains := func(ids []models.Identity, id uid.ID) bool {
```

Comment:
This seems like it could be slow for a big look-up, maybe we should sort or map the ids slice first.

Reply:
I'm not super happy with this being n^2, but even if you sorted it w/ n log(n) you still run into n^2 performance if the UID gets sorted to the end of the list. My thinking for not using a map originally was just that the slice would never be that big, but I guess I could see if there were thousands of IDs in a list and one wasn't found that it could be slow. I'm not sure how likely that is, but it's easy enough to change to a map.